addons/utm/views/utm_campaign_views.xml

```diff
@@ -54,11 +54,26 @@
         </field>
     </record>
 
+    <record id="quick_create_campaign_form" model="ir.ui.view">
+        <field name="name">utm.campaign.form.quick_create</field>
```

Comment:
guideline : align name and id. and replace '_' by '.' in name
```suggestion
    <record id="utm_campaign_view_form_quick_create" model="ir.ui.view">
        <field name="name">utm.campaign.view.form.quick.create</field>
```

Reply:
Thank you for the review :+1: